<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnalyzerWidget</class>
 <widget class="QWidget" name="AnalyzerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1127</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1,1,1,1">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>GroupBox</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Max noise amplitude</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="sbMaxNoiseAmplitude">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>100000</number>
            </property>
            <property name="value">
             <number>1000</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Noise density</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="dsbNoiseDensity">
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="minimum">
             <double>0.001000000000000</double>
            </property>
            <property name="maximum">
             <double>1.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.001000000000000</double>
            </property>
            <property name="value">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>230</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pbProcess">
            <property name="text">
             <string>Process</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pbStop">
            <property name="text">
             <string>Stop</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Wavelet result</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cbWaveletType"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextEdit" name="teThresholdsInfo"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QPushButton" name="pbSave">
            <property name="text">
             <string>Save</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pbLoad">
            <property name="text">
             <string>Load</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QPushButton" name="pbSaveInput">
            <property name="text">
             <string>Save input</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pbSaveNoised">
            <property name="text">
             <string>Save noised</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="AudioPlayerWidget" name="wInputSignalPlayer" native="true"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,1">
     <item>
      <widget class="QCustomPlot" name="wInputDifferencePlot" native="true"/>
     </item>
     <item>
      <widget class="AudioPlayerWidget" name="wNoisedIputSignalPlayer" native="true"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,1">
     <item>
      <widget class="QCustomPlot" name="wOutputDifferencePlot" native="true"/>
     </item>
     <item>
      <widget class="AudioPlayerWidget" name="wOutputSignalPlayer" native="true"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="teLog"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AudioPlayerWidget</class>
   <extends>QWidget</extends>
   <header>src/audio/Audioplayerwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
